python3/raygun4py/raygunmsgs.py

```diff
@@ -93,19 +98,19 @@ def set_tags(self, tags):
         if type(tags) is list:
             if not self.raygunMessage.details.get('tags'):
                 self.raygunMessage.details['tags'] = []
-            self.raygunMessage.details['tags'] += tags           
+            self.raygunMessage.details['tags'] += tags
         return self
 
     def set_request_details(self, request):
         if not request:
             return self
 
-        rg_request_details = http_utilities.build_wsgi_compliant_request(request)
+        rg_request_details = http_utilities.build_wsgi_compliant_request(
+            request)
```

Comment:
this line wrap ok?

Reply:
That does look pretty silly, no idea what autopep8 was thinking!